src/Cli/dotnet/commands/dotnet-workload/uninstall/WorkloadUninstallCommandParser.cs

```diff
@@ -12,7 +12,10 @@ internal static class WorkloadUninstallCommandParser
 
         public static readonly Option VerbosityOption = WorkloadInstallCommandParser.VerbosityOption;
         
-        public static readonly Option VersionOption = WorkloadInstallCommandParser.VersionOption;
+        public static readonly Option VersionOption = new Option<string>("--sdk-version", Workloads.Workload.Install.LocalizableStrings.VersionOptionDescription)
```

Comment:
Why is this needed? I think this is just a longer version of what we already had